chainer/links/caffe/caffe_function.py

```diff
@@ -302,6 +302,42 @@ def _setup_relu(self, layer):
         self.forwards[layer.name] = fw
         self._add_layer(layer)
 
+    @_layer('Scale', 'NONE')    # TODO: old name
```

Comment:
What should be set for ScaleLayer's old name, since I can't find it in caffe_pb2.py(rc3)'s `_V1LAYERPARAMETER_LAYERTYPE`?